modules/virt-snocaps-differences.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/about-virt.doc
+
+:_content-type: CONCEPT
+[id="virt-snocaps-differences_{context}"]
+= Single Node {product-title} differences
+
+You can install {VirtProductName} on a single-node cluster. See xref:../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[About {sno}] for more information. {sno-caps} does not support high availability, which results in the following differences:
+
+* xref:../nodes/pods/nodes-pods-priority.adoc#priority-preemption-other_nodes-pods-priority[Pod disruption budgets] are not supported.
+* xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration] is not supported.
+* Templates or virtual machines that use data volumes or storage profiles must not have the xref:../virt/live_migration/virt-configuring-vmi-eviction-strategy.adoc#virt-configuring-vmi-eviction-strategy[`evictionStrategy`] set.
+
+// Will add link as soon as I can determine how to add an xref to this topic, which appears to be in a repo outside OCP: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/deploying_openshift_data_foundation_on_single_node_openshift_clusters/index#doc-wrapper
+You can install the Logical Volume Manager on a single-node cluster by deploying {product-title} Data Foundation.
```

Comment:
Normally other docs referenced LVMO as `OpenShift Data Foundation Logical Volume Manager Operator`

Reply:
Thanks @akalenyu - Yes, I can document the assisted installer. From your Jira notes, it appears that the assisted installer is launched automatically in 4.12 for SNO installs. Is the installer self-documenting, or would it be beneficial to document the steps? If so, @jpeimer, can I get a test cluster so I could try out the installer or is there a video demo I can see?

Thanks also for the naming standard for LVMO.

Bob

Reply:
> From your Jira notes, it appears that the assisted installer is launched automatically in 4.12 for SNO installs

The assisted installer is a popular & simple way of provisioning an OpenShift cluster:
https://cloud.redhat.com/blog/using-the-openshift-assisted-installer-service-to-deploy-an-openshift-cluster-on-metal-and-vsphere
HPP related assisted installer presentation from 4.10:
https://docs.google.com/presentation/d/1ZyTt2TrWtzqhaMgKGDB2DSGT63TS1fAGzAbeYbo5gZ4/edit?usp=sharing

I don't think it would be necessary to document installation steps, instead we can describe how we provide things like LVMO out of the box when using assisted installer.

In the past I contributed some docs to the assisted installer repo describing how things behaved in 4.10:
https://github.com/openshift/assisted-service/pull/3610

Feel free if there are any other questions!

Reply:
Thanks @akalenyu - Please give the [(http://file.rdu.redhat.com/bgaydos/CNV-17539/virt/about-virt.html#virt-snocaps-differences_about-virt)conceptual module on differences] another look. 

Is there another Jira story for documenting the procedure for running the assisted installer in 4.12 and does it go beyond supporting only bare metal? I can see that topic fitting in next to the HPP topic in the Storage section. I can add a more descriptive summary of what the assisted installer does if there is no Jira story for how to use the installer in 4.12. Maybe we can meet briefly over video to ensure I'm not missing anything?

I am assuming the doc in the blog is obsolete as of the 4.12 assisted installer implementation, but if that's not true let me know.

Thanks for your help and patience!
Bob

Reply:
Hey again

We sure can meet and discuss this further.
I don't think we need to dive into the procedure of using assisted installer;
We should just document the storage it ships out of the box for the CNV workloads in different versions.
We can maybe link to a page about the assisted installer and that should be enough IMO